rustls/Cargo.toml

```diff
@@ -17,6 +17,36 @@ build = "build.rs"
 [features]
 default = []
 
+# NOTE: This unstable & undocumented feature is intended to help make it easier to configure
+# docs.rs to generate the documentation for all features where this is needed,
+# excluding any potentially problematic features such as:
+# - `unsafe-assume-single-core`
+# - FIPS (if added back to this fork someday in the future)
+#
+# Keeping some other features in this list to help make developer testing with
+# almost all features a little easier.
+#
+# Developer testing with all features enabled except for `unsafe-assume-single-core` can be
+# approximated by: --features unstable-api-all,critical-section
```

Comment:
extra-minor nit (may not be worth fixing at this point): replace this info for [portable-rustls] crate developer with reference such as this: "See further below for tips on crate developer testing"